jaxrs/src/main/java/com/cerner/beadledom/jaxrs/exceptionmappers/ThrowableExceptionMapper.java

```diff
@@ -0,0 +1,51 @@
+package com.cerner.beadledom.jaxrs.exceptionmappers;
+
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+
+import com.cerner.beadledom.jaxrs.models.JsonError;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Exception Mapper for all unhandled Exceptions thrown by the service.
```

Comment:
I don't think `Mapper` or the second `Exception` need to be capitalized.

Reply:
Updated the javadoc: https://github.com/cerner/beadledom/pull/45/commits/865a017a55ce93a2c50c539260b17f6bbcdfe110#diff-8fa995d7fc0ffa3c90069a9fc131c59bR16